[{'httpOnly': False, 'secure': False, 'path': '/', 'name': 'SSOLoginState', 'domain': '.weibo.cn', 'value': '1502377228'}, {'domain': '.weibo.cn', 'path': '/', 'name': 'SUHB', 'expiry': 1533913228.438377, 'secure': False, 'httpOnly': False, 'value': '05i6DGKnU50CCU'}, {'domain': '.weibo.cn', 'path': '/', 'name': 'SUB', 'expiry': 1533913228.437849, 'secure': False, 'httpOnly': True, 'value': '_2A250iAVcDeRhGeBM4lsT9ifNwj2IHXVUcqsUrDV6PUJbkdANLWjakW0E0Mn7ye4PFt4FIhy21KN41lWb1w..'}, {'domain': '.weibo.cn', 'path': '/', 'name': 'SCF', 'expiry': 1817737228.249946, 'secure': False, 'httpOnly': True, 'value': 'Atn8Br9K2StQnDlTaLeWKYq31FjD4APJkC-HOdXxiNTlqxaaJ-FcJQMymgTdQ73dO0iidd5AvW2CTOgCx11dZoM.'}, {'domain': '.weibo.cn', 'path': '/', 'name': '_T_WM', 'expiry': 1504969222.551311, 'secure': False, 'httpOnly': True, 'value': '1c8e1c31bd8bf81dff4518e8a14cd15f'}]
判断页面1成功 0失败  结果是=1
[<selenium.webdriver.remote.webelement.WebElement (session="92442821cd53b2f1e815519643f18b2f", element="0.49382825529995644-1")>, <selenium.webdriver.remote.webelement.WebElement (session="92442821cd53b2f1e815519643f18b2f", element="0.49382825529995644-2")>, <selenium.webdriver.remote.webelement.WebElement (session="92442821cd53b2f1e815519643f18b2f", element="0.49382825529995644-3")>, <selenium.webdriver.remote.webelement.WebElement (session="92442821cd53b2f1e815519643f18b2f", element="0.49382825529995644-4")>, <selenium.webdriver.remote.webelement.WebElement (session="92442821cd53b2f1e815519643f18b2f", element="0.49382825529995644-5")>, <selenium.webdriver.remote.webelement.WebElement (session="92442821cd53b2f1e815519643f18b2f", element="0.49382825529995644-6")>, <selenium.webdriver.remote.webelement.WebElement (session="92442821cd53b2f1e815519643f18b2f", element="0.49382825529995644-7")>, <selenium.webdriver.remote.webelement.WebElement (session="92442821cd53b2f1e815519643f18b2f", element="0.49382825529995644-8")>, <selenium.webdriver.remote.webelement.WebElement (session="92442821cd53b2f1e815519643f18b2f", element="0.49382825529995644-9")>, <selenium.webdriver.remote.webelement.WebElement (session="92442821cd53b2f1e815519643f18b2f", element="0.49382825529995644-10")>]
伦哥：好棒！第1次运行了
运行时间：2017-08-10 23:00:30
1
False
2
False
3
False
4
False
5
False
6
False
7
False
8
False
9
False
10
False
11
False
Traceback (most recent call last):
  File "my_repost.py", line 192, in <module>
    repost(driver1)
  File "my_repost.py", line 161, in repost
    browser.find_element_by_tag_name('body').send_keys(Keys.CONTROL + 'w')
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium/webdriver/remote/webdriver.py", line 409, in find_element_by_tag_name
    return self.find_element(by=By.TAG_NAME, value=name)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium/webdriver/remote/webdriver.py", line 791, in find_element
    'value': value})['value']
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium/webdriver/remote/webdriver.py", line 254, in execute
    response = self.command_executor.execute(driver_command, params)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium/webdriver/remote/remote_connection.py", line 464, in execute
    return self._request(command_info[0], url, body=data)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/selenium/webdriver/remote/remote_connection.py", line 487, in _request
    self._conn.request(method, parsed_url.path, body, headers)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/http/client.py", line 1083, in request
    self._send_request(method, url, body, headers)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/http/client.py", line 1128, in _send_request
    self.endheaders(body)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/http/client.py", line 1079, in endheaders
    self._send_output(message_body)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/http/client.py", line 911, in _send_output
    self.send(msg)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/http/client.py", line 854, in send
    self.connect()
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/http/client.py", line 826, in connect
    (self.host,self.port), self.timeout, self.source_address)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/socket.py", line 707, in create_connection
    raise err
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/socket.py", line 693, in create_connection
    sock = socket(af, socktype, proto)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/socket.py", line 134, in __init__
    _socket.socket.__init__(self, family, type, proto, fileno)
OSError: [Errno 23] Too many open files in system
